universal-login-relayer/lib/http/middlewares/errorHandler.ts

```diff
@@ -29,11 +30,11 @@ function getContentForException(err: Error) {
 }
 
 export function getCodeForException(err: Error) {
-  if (err instanceof ValidationFailed || err instanceof SanitizeError) {
+  if (err instanceof ValidationFailed || err instanceof SanitizeError || err instanceof ValidationError) {
```

Comment:
I don't like these names. ValidationError should be ValidationFailed (like before). When you import it use: 
```ts
import {ValidationFailed as CommonsValidationFailed} ... 
```